### PR TITLE
Remove fullscreen shortcut, add docs, bump version 0.1.2

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description

- Removed fullscreen shortcut for now, until we can address all the issues
- Added some additional documentation
- Bumped the version to 0.1.2 for release

Currently blocking this to wait on:
- #21 
- #26

### CODE REVIEW

**Review by stacyk — APPROVED** [review]
Looks good to me

**Review by jamesnw — APPROVED** [review]
👍

File: index.html [hunk 1]
```diff
@@ -178,10 +118,170 @@ <h2>Speaker Notes</h2>
...
+
+      <div><h2>Keyboard Shortcuts</h2></div>
+      <div>
+        <div slide-canvas>
+          <h3>Always available:</h3>
+          <ul>
+            <li><strong>command-k</strong>: Toggle control panel</li>
+            <li><strong>command-shift-enter</strong>: Start presentation (from first slide)</li>
+            <li><strong>command-enter</strong>: Resume presentation (from active slide)</li>
+            <li>
```
Comment (dvdherron) [inline index.html:140]: This command only worked for me in speaker view.

**Review by dvdherron — APPROVED** [review]
LGTM. Maybe a tiny note for Windows users to use `Ctrl` instead of `command` unless that's super obvious?

File: index.html (hunk 1 above)
Comment (mirisuzanne) [reply index.html:140]: Do you mean the grid view, or the script view or something else? Can you provide the steps to try and reproduce this?

**Comment by mirisuzanne** [comment]
> LGTM. Maybe a tiny note for Windows users to use `Ctrl` instead of `command` unless that's super obvious?

We should document it, yeah. Good catch.

File: index.html (hunk 1 above)
Comment (dvdherron) [reply index.html:140]: Ok, I maybe I was confused by this command `command-enter` being in the `Always Available` list. I wonder if it fits better in the `Active Presentation` list?

This happens in any view after I press `escape` while in a presentation. I understand that ends the presentation, but the last active slide remains highlighted with the dotted outline. This is how the `active` slides look during a presentation, so I thought by using this command I could re-enter the presentation at that same highlighted slide, but it starts the presentation from the beginning.

Comment (mirisuzanne) [reply index.html:140]: It's hard to know where the bug is there, since I can't reproduce it on my end. Maybe we can screen-share and debug it together?

Comment (dvdherron) [reply index.html]: I don't think this is still relevant now that the `esc` shortcut is gone. `command-enter` works as expected now.

**Review by dvdherron — APPROVED** [review]
LGTM

**Pull request merged**